stream-platform/src/main/java/io/camunda/zeebe/stream/impl/ProcessingStateMachine.java

```diff
@@ -200,18 +200,21 @@ public ProcessingStateMachine(
 
   private void skipRecord() {
     notifySkippedListener(currentRecord);
-    inProcessing = false;
+    markProcessingCompleted();
     actor.submit(this::readNextRecord);
     metrics.eventSkipped();
   }
 
   void readNextRecord() {
```

Comment:
🔧 Can we remove the method then?

Reply:
I think, that makes sense now. I will remove this.